core/modules/syncer.js

```diff
@@ -633,6 +633,10 @@ DeleteTiddlerTask.prototype.run = function(callback) {
 		}
 		// Remove the info stored about this tiddler
 		delete self.syncer.tiddlerInfo[self.title];
+		if($tw.boot.files){
+			// Remove the tiddler from $tw.boot.files
+			delete self.boot.files[self.title];
```

Comment:
I believe this should be `$tw.boot.files` ? otherwise it just generates a slew or fatal errors.

Reply:
Good catch, thanks.